thirdparty/libbacktrace/config.h

```diff
@@ -95,12 +102,21 @@
 /* Define to 1 if you have the <sys/types.h> header file. */
 #define HAVE_SYS_TYPES_H 1
 
+/* Define to 1 if you have the <tlhelp32.h> header file. */
+#define HAVE_TLHELP32_H 1
+
 /* Define to 1 if you have the <unistd.h> header file. */
 #define HAVE_UNISTD_H 1
 
+/* Define to 1 if you have the <windows.h> header file. */
+#define HAVE_WINDOWS_H 1
+
 /* Define if -lz is available. */
 #define HAVE_ZLIB 1
 
+/* Define if -lzstd is available. */
+/* #undef HAVE_ZSTD */
```

Comment:
Technically we have it so we could make it work, but none of the files we include make use of it.